src/main/java/com/peerrequest/app/data/Category.java

```diff
@@ -0,0 +1,124 @@
+package com.peerrequest.app.data;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.persistence.UniqueConstraint;
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+import java.util.Objects;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.Hibernate;
+import org.hibernate.annotations.TimeZoneStorage;
+import org.hibernate.annotations.TimeZoneStorageType;
+
+/**
+ * Represents a Category.
+ */
+@Builder
+@AllArgsConstructor
+@Entity
+@Table(name = "category", uniqueConstraints = {
+    @UniqueConstraint(name = "uc_category_name_label", columnNames = {"name", "label"})
+})
+public class Category {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    @Column(name = "id", nullable = false)
+    @Getter
+    private Long id;
+    @Column(name = "researcher_id", nullable = false, length = 40)
+    @Getter
+    private String researcherId;
+    @Column(name = "name", nullable = false)
+    @Getter
+    @Setter
+    private String name;
+    @Enumerated(EnumType.STRING)
+    @Column(name = "label", nullable = false)
+    @Getter
+    @Setter
+    private CategoryLabel label;
+    @Column(name = "deadline", nullable = false)
+    @TimeZoneStorage(TimeZoneStorageType.NORMALIZE_UTC)
+    @Getter
+    @Setter
+    private ZonedDateTime deadline;
+    @Column(name = "min_score", nullable = false)
+    @Getter
+    @Setter
+    private float minScore;
+    @Column(name = "nax_score", nullable = false)
+    @Getter
+    @Setter
+    private float maxScore;
+
```

Comment:
scoreStepSize is missing

Reply:
yes, I do not see any value in having it. What do you think?

Reply:
For the different conventions of different conferences. For example, one conference may allow a rating form 0 to 1 with 0.1 steps, the next may only allow ratings from 0 to 1 with 0.25 steps.